### ISSUE TITLE
The main script has some minor invalid comparison errors

### ISSUE THREAD

**Issue by jgabaut** [jgabaut/amboso#64]
**Describe the bug**
With `1.9.1`, `amboso`'s `source_amboso_api()` function was rewritten in an attempt to move to a POSIX shell. This backfired as there's some errors in the reformat of the function.

**To Reproduce**
1. Run `anvil -v`
2. Error was:

```console
/usr/local/bin/anvil: line 69: [: : integer expression expected
/usr/local/bin/anvil: line 81: [: : integer expression expected
```

**Expected behavior**
The mentioned check is done correctly.

 - Version `>= 1.9.1`

**Comment by jgabaut** [comment]
It seems the refactor was checking for `quiet_flag` without defining it.
Should be fixed with `1.9.2`.

**Comment by jgabaut** [comment]
It seems there's still an instance of the same problem:

```console
/usr/local/bin/anvil: line 74: [: : integer expression expected
```

**Comment by jgabaut** [comment]
Should be closed as of `1.9.4`.
